Formula/dsvpn.rb

```diff
@@ -0,0 +1,57 @@
+class Dsvpn < Formula
+  desc "Dead Simple VPN"
+  homepage "https://github.com/jedisct1/dsvpn"
+  url "https://github.com/jedisct1/dsvpn.git",
+      :tag      => "0.1.4",
+      :revision => "3b99d2ef6c02b2ef68b5784bec8adfdd55b29b1a"
+  head "https://github.com/jedisct1/dsvpn.git"
+
+  def install
+    system "make"
+    bin.install "dsvpn"
```

Comment:
Use `system "make", "install", "PREFIX=#{prefix}"` instead

Reply:
This is done. I had to create the `sbin` directory first though due to work around some wonkiness in the Makefile.

On that note, dsvpn lives in sbin and has a link /usr/local/sbin/dsvpn. Should I warn users or provide a message they need /usr/local/sbin in the path or should this be tacit knowledge when installing dsvpn? Or maybe dsvpn page should provide instructions on this.

Thanks.

Reply:
Sorted. I have now added a caveats explaining how to run dsvpn as root and I spelled out the directory name with "sbin", so it's absolutely clear how to run it.

Thanks.